backend/pkg/rest.utils/paging.go

```diff
@@ -37,6 +37,14 @@ var (
 	)
 )
 
+func MsgQueryParmInvalid(param, name string) string {
+	return fmt.Sprintf("invalid %s query: \"%s\"", param, name)
+}
+
+func MsgQueryParmLimit(name string) string {
+	return fmt.Sprintf("invalid %s query: value must be a non-zero positive integer", name)
+}
```

Comment:
This could be tidied up a little:
```suggestion
func ErrQueryParmInvalid(param, name string) error {
	return fmt.Errorf("invalid %s query: \"%s\"", param, name)
}

func ErrQueryParmLimit(name string) error {
	return fmt.Errorf("invalid %s query: value must be a non-zero positive integer", name)
}
```